wind.go

```diff
@@ -66,7 +67,7 @@ func (w *Window) initHeadless(clone *Window) *Window {
 	w.tag.w = w
 	w.taglines = 1
 	w.tagsafe = false
-	w.tagexpand = true
+	w.tagexpand = false
```

Comment:
Why this change? It seems unrelated to your CL and modifies the UI? Maybe you meant `w.tabexpand = false`?

Reply:
You're correct, I fixed this in a later commit.